neutron/include/scipp/neutron/conversions.h

```diff
@@ -17,5 +20,26 @@ constexpr auto energy_to_tof = [](auto &coord, const auto &c) {
 constexpr auto wavelength_to_q = [](auto &coord, const auto &c) {
   coord = c / coord;
 };
+constexpr auto tof_to_energy_transfer = [](auto &coord, const auto &scale,
```

Comment:
can we better document the args in terms of the physics? For example `scale` is `m * L * L` (I assume)

Reply:
It is something like `0.5 * m_N * "J-to-meV" / (s * s) * L1 * L1`, or `- L2 * L2`, depending on the case.